node/resolvers/checkout/attachmentsHelper.ts

```diff
@@ -137,6 +138,7 @@ const addOptionsRecursive = async (
       continue
     }
 
+    // eslint-disable-next-line no-await-in-loop
```

Comment:
in these loop cases, you can follow this [documentation](https://eslint.org/docs/latest/rules/no-await-in-loop)

would be like this:

```ts
const addOptionsRecursive = async (
  items: OptionItems,
  assemblyId: string,
  orderForm: OrderForm,
  oldItems: OrderFormItem[],
  checkout: Context['clients']['checkout']
  // eslint-disable-next-line max-params
) => {
  const recentlyAdded = getNewItemsOnly(oldItems, orderForm.items)
  const results = []

  for (const item of items) {
    const parentItem = findRecentlyAddedParent(
      recentlyAdded,
      item.id!.toString(),
      assemblyId
    )

    const parentIndex =
      parentItem &&
      orderForm.items.findIndex(propEq('uniqueId', parentItem.uniqueId))

    if (parentIndex == null || parentIndex < 0) {
      continue
    }

    results.push({
      checkout,
      itemIndex: parentIndex,
      options: item.options,
      orderForm,
      oldItems,
    })
  }

  Promise.all(results)
}```